src/libtexture/texturesys.cpp

```diff
@@ -2208,7 +2208,7 @@ TextureSystemImpl::sample_bilinear(
                     // iteration, as long as we aren't using mirror wrap mode!
                     if (i == 0 || tile_s == 0 || noreusetile) {
                         id.xy(tile_edge[S0 + i], tile_edge[T0 + j]);
-                        bool ok = find_tile(id, thread_info);
+                        bool ok = find_tile(id, thread_info, sample == 0);
```

Comment:
Why is it ok for only `sample=0` to mark as used? Can't different samples probe into different tiles?